praw/models/reddit/emoji.py

```diff
@@ -127,11 +169,24 @@ def __iter__(self):
                 self._reddit, self.subreddit, emoji_name, _data=emoji_data
             )
 
-    def add(self, name, image_path):
+    def add(
+        self,
+        name,
+        image_path,
+        post_flair=True,
+        user_flair=True,
+        mod_only=False,
```

Comment:
Here also, could these parameters be alphabetized? (and in the doc string as well)

Reply:
Also noted and taken care of.